### PR TITLE
apply reviews to AutoFeat documentation

### PR DESCRIPTION
**Pull request by Hiba-Alili**

### CODE REVIEW

File: src/docs/PML/PMLUserGuide.adoc [hunk 1]
```diff
@@ -1566,13 +1566,13 @@ image::NEW_MAAS_DL_MNIST_Workflow_Example.PNG[align=center]
...
+Before going for the modelling, we require to perform various tasks for data preparation. Encoding categorical data is one of the most crucial tasks. In real life, data commonly come with categorical string values and most of the machine learning models perform mathematical operations. However, the harsh truth is that mathematics is totally dependent on numbers. As a matter of fact, we can say that most of the machine learning models only accept numerical variables (generally floats or integers) and not strings. Then, preprocessing and encoding the categorical variables become a crucial step to convert these variables into numbers that can help in predicting the results in a machine learning task.
```
Comment (dianajlailaty) [inline src/docs/PML/PMLUserGuide.adoc:1571]: Before starting the modelling phase, it is required to perform various tasks for data preparation.

File: src/docs/PML/PMLUserGuide.adoc [hunk 2]
```diff
@@ -1585,13 +1585,13 @@ AutoFeat currently supports the following encoding methods:
...
+As we already mentioned, the performance of ML algorithms depends on how categorical variables are encoded. The results produced by the model vary depending on the used encoding techniques. Thus, the hardest part of categorical encoding can sometimes be finding the right categorical encoding method.
 
-There are numerous research papers and studies dedicated to analyzing the performance of categorical encoding approaches to different datasets. Based on the common factors shared by the datasets using the same encoding method, we have implemented an algorithm for finding the method best suited for your data. This can help the data scientist to start encoding smarter.
+There are numerous research papers and studies dedicated to the analysis of the performance of categorical encoding approaches applied to different datasets. Based on the common factors shared by the datasets using the same encoding method, we have implemented an algorithm for finding the best suited method for your data.
 
 To access the AutoFeat page, please follow the steps below:
 
-Open the link:https://try.activeeon.com/studio[Studio Portal].
+Open the link:https://trydev2.activeeon.com/studio[Studio Portal].
```
Comment (dianajlailaty) [inline src/docs/PML/PMLUserGuide.adoc:1594]: Should it be trydev2 in the documentation? The reader is not supposed to have access to trydev2

File: src/docs/PML/PMLUserGuide.adoc [hunk 3]
```diff
@@ -1626,15 +1626,15 @@ Execute the workflow by setting the different workflow's variables as described
 | Int (-1 means preview all the rows)
 |===
 
-Open the link:https://try.activeeon.com/automation-dashboard/#/portal/workflow-execution[Workflow Execution Portal].
+Open the link:https://trydev2.activeeon.com/automation-dashboard/#/portal/workflow-execution[Workflow Execution Portal].
```
Comment (dianajlailaty) [inline src/docs/PML/PMLUserGuide.adoc:1629]: same here for trydev2

File: src/docs/PML/PMLUserGuide.adoc (hunk 2 above)
Comment (Hiba-Alili) [reply src/docs/PML/PMLUserGuide.adoc:1594]: I think also. Would you confirm @mboussaa? Thanks :)

Comment (mboussaa) [reply src/docs/PML/PMLUserGuide.adoc:1594]: yes try.activeeon.com

File: src/docs/PML/PMLUserGuide.adoc (hunk 3 above)
Comment (mboussaa) [reply src/docs/PML/PMLUserGuide.adoc:1629]: @Hiba-Alili let's use try for now

**Review by mboussaa — APPROVED** [review]

**Pull request merged**